packages/core/nuxt-theme-module/theme/helpers/errors/authentication.ts

```diff
@@ -0,0 +1,21 @@
+export const authErrors = (context, email) => {
```

Comment:
wait... what??? why do you need such a mapping? use just translation keys within integration:

eg:
```
"There is already an existing customer with this email": "There is already an existing customer with this email {email}"
```

and in the code
```
$t(response.message, { email }) // vue i18n already supports interpolation
```

I know you need interpolation for the keys too, but maybe you can recognize this kind of error by using eg. an error code or something else? Clearly, look into i18n docs I'm pretty sure you can solve this